Tests/Functional/Doctrine/Phpcr/RedirectRouteTest.php

```diff
@@ -17,15 +17,28 @@
 
 class RedirectRouteTest extends BaseTestCase
 {
-    const ROUTE_ROOT = '/test/redirectroute';
+    const ROUTE_ROOT = '/test/redirectroute-functional';
 
-    public function setUp()
+    protected function setUp()
     {
-        parent::setUp();
-        $this->db('PHPCR')->createTestNode();
+        $this->makeSureTestRootExists();
         $this->createRoute(self::ROUTE_ROOT);
     }
 
+    /**
+     * Cleans up the created nodes in the database.
+     */
+    protected function tearDown()
```

Comment:
Why don't we move the body of this method into the `BaseTestCase` ? (seems to be a duplicate of the `tearDown` in the RedirectControllerTest)